frontend/device/input.lua

```diff
@@ -240,6 +241,12 @@ function Input:adjustTouchTranslate(ev, by)
     end
 end
 
+function Input:adjustKindleOasisOrientation(ev)
```

Comment:
Howerver, kindle device does not know anything about the details of event code and structure so it should be processed in low-level module.